about.md

```diff
@@ -3,4 +3,23 @@ title: L'intention
 permalink: /about/
 ---
 
-:-)
+L'équipe d'[Étalab][] se réunit pour produire un calendrier de l'Avent de l'[Open Data][]. Ce calendrier se veut être une rétrospective de l'année 2017.
```

Comment:
AFAIK, no accent [because of the arrêté](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000031194455&categorieLien=id).